### PR TITLE
Fix humansegv2 after #1629 changes

### PR DESCRIPTION
**Pull request by piotrekobi**
Adds the following fixes in response to #1629:
- Re-enables humansegv2 in bf16 mode (first added in #1529)
- Added tenssort module to requirements.txt
- Added exception handling to pycuda import. The library requires CUDA to be installed. This change allows to run the model on machines without CUDA.

### CODE REVIEW

**Comment by sfraczek** [comment]
LGTM. I don't have review permissions.

**Comment by piotrekobi** [comment]
@yghstill @chalsliu Please review

**Comment by piotrekobi** [comment]
@yaomichael  Please help get this merged. 
Without these changes, we can't run the humansegv2 model in our local validation.
@yghstill @chalsliu

**Review by yeliang2258 — APPROVED** [review]
LGTM

**Review by chalsliu — APPROVED** [review]
LGTM

File: inference/python_api_test/test_int8_model/backend/paddle_inference.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ def __init__(
                 config.enable_mkldnn()
                 if precision == "int8":
                     config.enable_mkldnn_int8({"conv2d", "depthwise_conv2d", "pool2d", "transpose2", "elementwise_mul"})
+                if precision == "bf16":
+                    config.enable_mkldnn_bfloat16()
 
                 if precision == "bf16":
                     config.enable_mkldnn_bfloat16()
```
Comment (sfraczek) [inline inference/python_api_test/test_int8_model/backend/paddle_inference.py:78]: Duplicate @piotrekobi

**Pull request merged**